packages/core/src/browser/dialogs.ts

```diff
@@ -458,11 +458,18 @@ export class ConfirmSaveDialog extends AbstractDialog<boolean | undefined> {
 
 }
 
+export enum confirmExitWithOrWithoutSavingResult {
```

Comment:
I wonder if we should not move this method over to `common-frontend-contribution.ts`. It only seems to be invoked from there and it seems to me we could simplify the code if we did.

Reply:
moved it. seems logical and collected.